client/src/components/CarouselDots/CarouselDots.test.js

```diff
@@ -1,16 +1,46 @@
 import React from "react";
-import { shallow, configure } from "enzyme";
+import { shallow, configure, mount } from "enzyme";
 import CarouselDots from "./CarouselDots.js";
 import Adapter from "enzyme-adapter-react-16";
 configure({ adapter: new Adapter() });
 
 // We will describe a block of tests
 describe("CarouselDots component", () => {
   // we will write one individual test
-  it("should render as expected", () => {
+  it("Component should render the number of dots supplied in the properties", () => {
+
     // Shallow rendering renders a component without rendering any of its children
-    const component = shallow(<CarouselDots />);
-    // We create an assertion within the test that checks if our component renders our name prop
-    // expect(component.contains('Your name')).toBe(true)
+    const component = mount(<CarouselDots
+    numberOfDots={3}
+    activeLocation={1}
+    onStatusChange={jest.fn()}/>);
+    console.log(component.debug())
```

Comment:
Can you remove this `console.log`?

Reply:
Can you send me a copy of the errors?  I get this: 
    ✓ Component should render the number of dots supplied in the properties (35ms)
    ✓ Component should switch from dot 1 to dot 2 upon clicking 2nd dot (7ms)
    ✓ Component should render border when prop `border` is set  (3ms)